homeassistant/components/iperf3/sensor.py

```diff
@@ -16,8 +20,16 @@
 ATTR_REMOTE_PORT = "Remote Port"
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info):
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
```

Comment:
The discovery_info is currently a list and not a dict.

Reply:
I have moved this out, and created a separate PR #64204
Was there anything else to sort out in this PR?

Reply:
I think that was it. 👍